orchestra/static/orchestra/dashboard/dashboard.html

```diff
@@ -1,5 +1,6 @@
 <section class="wrapper">
   <div tasktable="{label: 'Active', tasks: vm.orchestraTasks.activeTasks(), newTasks: true}"></div>
   <div tasktable="{label: 'Pending', tasks: vm.orchestraTasks.pendingTasks()}"></div>
+  <div tasktable="{label: 'Paused', tasks: vm.orchestraTasks.pausedTasks()}"></div>
```

Comment:
at some point we discussed this being collapsed by default, along with completed projects. what do you think?